src/index.html

```diff
@@ -12,6 +12,54 @@
+            <stop  offset="0.8945" style="stop-color:#797B5A"/>
+            <stop  offset="0.9469" style="stop-color:#6E7050"/>
+            <stop  offset="0.989" style="stop-color:#5D5F42"/>
+            <stop  offset="1" style="stop-color:#57593D"/>
+          </linearGradient>
+          <linearGradient id="SVGID_1_" gradientUnits="userSpaceOnUse" x1="15.2693" y1="79.4649" x2="15.2693" y2="36.0682" gradientTransform="matrix(0.9986 -0.0523 0.0523 0.9986 14.3576 -33.8624)">
+            <stop  offset="0" style="stop-color:#57593D"/>
+            <stop  offset="0.1826" style="stop-color:#5B5D41"/>
+            <stop  offset="0.3808" style="stop-color:#686A4C"/>
+            <stop  offset="0.5858" style="stop-color:#7E805E"/>
+            <stop  offset="0.6167" style="stop-color:#828461"/>
+            <stop  offset="0.8209" style="stop-color:#80825F"/>
+            <stop  offset="0.8945" style="stop-color:#797B5A"/>
+            <stop  offset="0.9469" style="stop-color:#6E7050"/>
+            <stop  offset="0.989" style="stop-color:#5D5F42"/>
+            <stop  offset="1" style="stop-color:#57593D"/>
+          </linearGradient>
+          <linearGradient id="SVGID_2_" gradientUnits="userSpaceOnUse" x1="39.1793" y1="12.752" x2="39.1793" y2="-16.7309" gradientTransform="matrix(0.9986 0.0523 -0.0523 0.9986 -16.7667 15.0681)">
+            <stop  offset="0" style="stop-color:#57593D"/>
+            <stop  offset="0.1826" style="stop-color:#5B5D41"/>
+            <stop  offset="0.3808" style="stop-color:#686A4C"/>
+            <stop  offset="0.5858" style="stop-color:#7E805E"/>
+            <stop  offset="0.6167" style="stop-color:#828461"/>
+            <stop  offset="0.8209" style="stop-color:#80825F"/>
+            <stop  offset="0.8945" style="stop-color:#797B5A"/>
+            <stop  offset="0.9469" style="stop-color:#6E7050"/>
+            <stop  offset="0.989" style="stop-color:#5D5F42"/>
+            <stop  offset="1" style="stop-color:#57593D"/>
+          </linearGradient>
+          <linearGradient id="SVGID_3_" gradientUnits="userSpaceOnUse" x1="4.2318" y1="14.5491" x2="4.2318" y2="-14.9338" gradientTransform="matrix(0.9986 -0.0523 0.0523 0.9986 17.8137 16.0558)">
+            <stop  offset="0" style="stop-color:#57593D"/>
+            <stop  offset="0.1826" style="stop-color:#5B5D41"/>
+            <stop  offset="0.3808" style="stop-color:#686A4C"/>
+            <stop  offset="0.5858" style="stop-color:#7E805E"/>
+            <stop  offset="0.6167" style="stop-color:#828461"/>
+            <stop  offset="0.8209" style="stop-color:#80825F"/>
+            <stop  offset="0.8945" style="stop-color:#797B5A"/>
+            <stop  offset="0.9469" style="stop-color:#6E7050"/>
+            <stop  offset="0.989" style="stop-color:#5D5F42"/>
+            <stop  offset="1" style="stop-color:#57593D"/>
+          </linearGradient>
         <symbol id="icon-divider" viewBox="0 0 31 93">
```

Comment:
Position of the linearGradient definitions was causing gradients to fail to render on Firefox!